app/backend/src/couchers/models.py

```diff
@@ -958,15 +958,15 @@ class HostRequest(Base):
     @hybrid_property
     def can_write_reference(self):
         return (
-            (self.status == HostRequestStatus.confirmed)
+            (self.status == HostRequestStatus.confirmed | self.status == HostRequestStatus.accepted)
```

Comment:
Read up on the priority of bitwise operations versus comparators. (You need parentheses here or switch to `or`)

Reply:
Yes... I should've known that if I'd thought a bit harder...